docs/docs/advanced-use/system-administration.md

```diff
@@ -284,35 +284,51 @@ To this:
 ```json
 [
  {
+  "id": null,
   "feature": {
    "id": 9422,
    "name": "first_feature",
+   "created_date": null,
+   "description": null,
+   "initial_value": null,
+   "default_enabled": null,
    "type": "STANDARD"
   },
   "feature_state_value": null,
+  "environment": null,
+  "identity": null,
+  "feature_segment": null,
   "enabled": false
  },
  {
+  "id": null,
   "feature": {
    "id": 9423,
    "name": "second_feature",
+   "created_date": null,
+   "description": null,
+   "initial_value": null,
+   "default_enabled": null,
    "type": "STANDARD"
   },
   "feature_state_value": null,
+  "environment": null,
+  "identity": null,
+  "feature_segment": null,
   "enabled": false
  }
 ]
 ```
 
 :::info
 
-All fields mentioned are already excluded in the response generated by the [Edge API](./edge-api.md).
+All fields mentioned are not part of the response generated by the [Edge API](./edge-api.md).
 
 :::
 
 ### `/api/v1/identities`
 
-Following fields will be removed from the response:
+The following fields will always be Null:
```

Comment:
This isn't quite true, right? `traits` will not be `null`

Reply:
Yeah, good catch